browser/resources/settings/brave_privacy_page/brave_tor_subpage.html

```diff
@@ -0,0 +1,65 @@
+<style include="settings-shared md-select">
+  img {
+    width: 100%;
+  }
+
+  #torBridgesControls {
+    padding: 0 var(--cr-section-padding);
+  }
+
+  #requestedBridges,
+  #providedBridges {
+    width: 90%;
+    min-height: 70px;
+    height: auto;
+    resize: vertical;
+  }
+
+  textarea {
+    white-space: pre;
+    overflow-wrap: normal;
+    overflow-x: scroll;
+  }
+</style>
+<div id="torBridgesControls">
```

Comment:
Can you please attach screenshots of this (and the dialog) to the PR description? This is for the purpose of Design (and QA) review.

Reply:
Video uploaded. (See Test plan section)